core/src/main/java/org/apache/iceberg/SchemaUpdate.java

```diff
@@ -476,4 +476,6 @@ public Type primitive(Type.PrimitiveType primitive) {
     newFields.addAll(adds);
     return Types.StructType.of(newFields);
   }
+
+  //TODO: updateEvent
```

Comment:
I think we can add update events individually.

Reply:
Makes sense